add callable values (#3748)
EasyCorp/EasyAdminBundle: doc/fields.rst

The main limitation of unmapped fields is that they are not sortable because
they cannot be included in the Doctrine query.

Virtual Fields
~~~~~~~~~~~~~~~

> You can define an virtual fields that's not exists in entity but which need to be calculated::

Reviewer: ```suggestion
You can define an virtual field that doesn't exist in the entity but needs to be calculated::
```

    public function configureFields(string $pageName): iterable
    {
        yield MoneyField::new('balance'')
            ->setVirtual(true)

>             ->setValue(fn(Customer $customer) => $this->clientService->getBalance($customer))

Reviewer: This syntax is only supported on PHP 8

Author: arrow function was introduced in php 7.4. some examples in docs also with arrow func

Reviewer: Oh you are right, thanks
